### ISSUE TITLE
Delete icon should be disabled when story only contains a single page

### ISSUE THREAD

**Issue by wassgha** [GoogleForCreators/web-stories-wp#783]
## Bug Description

Having a one-page story and deleting its only page has no effect. 

## Expected Behaviour

We can go multiple ways about this. Pressing the button should either:
- Empty out the page and reset its background/properties
- Actually delete the page and show an empty state
- Not delete the page and show a snackbar ("Can't delete the only page left")

## Steps to Reproduce

- Press the delete page button on a story that only has one page

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by miina** [comment]
See also the previous conversation here on the same topic with the conclusion that the last page can't (shouldn't) be deleted: https://xwp.slack.com/archives/CCCAUAH6F/p1576261754010100

**Comment by samitron7** [comment]
We do not allow deleting the last page. We should disable (gray out) the delete page icon.

From Pascal:
Note: For accessibility reasons, it should probably only appear disabled, but still be able to receive focus. Otherwise it won't get announced to screen readers.

**Comment by miina** [comment]
@barklund Perhaps this could be done within #6204 since the icon needs to be adjusted anyway? Or is that issue just an icon swap and better to keep it separately?
